src/content/posts/2024/10/29/tuist-ambassadors.mdx

```diff
@@ -0,0 +1,50 @@
+and helped us reach communities that we wouldn't have reached otherwise.
+Tuist wouldn't be what it is today without them.
+
+So we want to give back to the community by **recognizing these people and their contributions.**
+And we are doing so by introducing the Tuist Ambassadors program.
+
+## What is the Tuist Ambassadors program?
+
+Once a year, **we'll select a small group of Tuist Ambassadors** who have been nominated by the community for representing Tuist, building bridges between the core team and all Tuisters, and leading by example in their own communities.
+
+Tuist Ambassadors become recognized leaders and advisors in the Tuist community and strong advocates for the values of the project. They are not only Tuist experts: they lead, support, and inspire others. They play a pivotal role in advancing our shared goals.
+
+## What are the benefits of being a Tuist Ambassador?
+
+- **Visibility and recognition:** You'll be recognized as a Tuist Ambassador on the Tuist website and in the project's communication channels. You'll get a special role in the community forum.
+- **One-of-a-kind Tuist swag:** Ambassadors get our iconic Ambassador jacket and a custom patch for each Ambassador batch year.
+- **Access and influence:** Ambassadors are more tightly integrated into the shaping process and have the ability to provide early feedback before releases are out. They also join periodic calls with the Tuist team to discuss roadmaps and provide feedback.
+- **Amplify Budget:** Tuist supports Ambassadors who speak at external events, covering eligible expenses and providing other logistical support where appropriate.
+
+## What do you need to do to become a Tuist Ambassador?
+
+The following criteria will be used to evaluate potential Tuist Ambassadors:
+
+- Highly knowledgeable about Tuist - recognized expertise in the Tuist tech and able to articulate the benefits and vision behind Tuist.
+- Have contributed to the Tuist project in a meaningful way, and/or is an active user of Tuist within their organization.
+- Technical and direct tone, not "salesy."
+- Do not come across to other community members as pushing another product, service offering, or agenda.
+- Regularly produce technical content for the community such as presenting on community calls or contributing a blog post (at least once every six months).
+- Represent Tuist at public events such as meetups or industry events (at least once every six months).
+- Actively help other Tuisters in the [community forum](https://community.tuist.io)
+- Provide feedback on Tuist features and releases
+- Be a model Tuist Ambassador: respect our code of conduct, especially when critical of Tuist or defending Tuist against others' criticism; remain respectful, constructive, and fair
+
+## How to nominate someone?
+
+The nominations for 2025 are now open. If you know someone who you think would be a great Tuist Ambassador, please fill out the [nomination form](LINK TO THE COMMUNITY TOPIC).
```

Comment:
I plan to use [this Discourse plugin](https://meta.discourse.org/t/custom-wizard-plugin/73345) and post a topic in the forum from where people can privately nominate.